ln-gateway/src/bin/ln_gateway.rs

```diff
@@ -132,9 +132,10 @@ async fn htlc_accepted_hook(
 ) -> Result<serde_json::Value, Error> {
     let htlc_accepted: HtlcAccepted = serde_json::from_value(value)?;
     let preimage = gw_rpc(plugin, htlc_accepted).await?;
+    let pk = preimage.to_public_key()?;
```

Comment:
While this preimage is a public key in practice we don't care in the Gateway and should directly serialize the bytes as hex instead.

Reply:
@okjodom What about this one? Unless I'm confused, should we address already?

Reply:
Marked for follow up 👍🏿